e2e/stack_test.go

```diff
@@ -282,48 +277,273 @@ func depositE2E(t *testing.T, stack *e2e.StackConfig) {
 	receipt, err := l1Client.Client.TransactionReceipt(stack.Ctx, depositTx.Hash())
 	require.NoError(t, err, "deposit tx receipt")
 	require.NotNil(t, receipt, "deposit tx receipt")
-	require.NotZero(t, receipt.Status, "deposit tx reverted") // receipt.Status == 0 -> reverted tx
+	require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status, "deposit tx reverted")
 
 	depositLogs, err := stack.L1Portal.FilterTransactionDeposited(
-		&bind.FilterOpts{
-			Start:   0,
-			End:     nil,
-			Context: stack.Ctx,
-		},
-		nil, // from any address
-		nil, // to any address
-		nil, // any event version
+		createL1FilterOpts(stack.Ctx),
+		[]common.Address{user.Address},
+		[]common.Address{user.Address},
+		[]*big.Int{big.NewInt(0)},
 	)
 	require.NoError(t, err, "configuring 'TransactionDeposited' event listener")
 	if !depositLogs.Next() {
-		require.FailNowf(t, "finding deposit event", "err: %w", depositLogs.Error())
+		require.FailNowf(t, "finding deposit event", "err: %v", depositLogs.Error())
+	}
+	t.Log("Monomer can send user deposit txs on L1")
```

Comment:
I think Monomer hasn't really done anything up to this point. It's a useful signpost in the deposit lifecycle, but should we reserve these celebratory logs for Monomer's work?

Reply:
Yeah good point, I'll remove the logs that reference non-monomer work